.github/workflows/publish-docker.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+    tags:
+      - "v*"
```

Comment:
what does this tag in particular do?

Reply:
This workflow also runs on a tag starting with "v".

To improve it a bit, we could change it to:

```yaml
tags:
  - "v[0-9]+.[0-9]+.[0-9]+"
``` 

Then the workflow only runs on tags with the `major.minor.patch` semantic release version.